lib/entities/parse_address_from_domain.dart

```diff
@@ -26,6 +27,14 @@ class AddressResolver {
 
   Future<ParsedAddress> resolve(String text, String ticker) async {
     try {
+      if(text.contains('@')){
```

Comment:
Please tighten up this up to not check if the string includes any periods `.` since OpenAlias also supports emails, eg for XMR justin@ehrenhofer.org. It should check `justin@ehrenhofer` for an FIO address, not `justin@ehrenhofer.org`.

Reply:
done